ts-scripts/generateExternalTokens.ts

```diff
@@ -112,26 +109,20 @@ const formatApiTokenMetadata = async (
 
     if (denom.startsWith('ibc/')) {
       const ibcToken = await fetchIbcTokenMetaData(denom, Network.MainnetSentry)
-      const denomFromBank = getSupplyDenom(
```

Comment:
no need to override denom with uppercase denom, since the `denom` variable is already upper case now